internal/cloudapi/v2/v2.go

```diff
@@ -427,6 +422,22 @@ func (h *apiHandlers) PostCompose(ctx echo.Context) error {
 	})
 }
 
+func imageTypeFromApiImageType(it ImageTypes) string {
+	switch it {
+	case ImageTypes_aws:
+		return "ami"
+	case ImageTypes_gcp:
+		return "vhd"
+	case ImageTypes_azure:
+		return "vhd"
+	case ImageTypes_edge_commit:
+		return "rhel-edge-commit"
```

Comment:
Does this still work for 8.5? Did we add an alias, or did we actually rename it to `edge-commit`, etc? @achilleas-k ?

Reply:
The canonical name is `edge-commit` but it's also still registered under the alias `rhel-edge-commit`.

Reply:
Ah shall I change that?

Reply:
`edge-commit` and `edge-installer` wont work in 8.4 but `rhel-edge-*` work in 8.4 and 8.5 (and 8.6 if we decide to keep the aliases).  So maybe we should keep the rhel prefix internally.

Reply:
I would say expose it always as `edge-commit`, and then internally we can translate to `rhel-edge-commit` (which we can drop eventually if we no longer support 8.4).

@achilleas-k probably should keep the alias in 8.X, backwards compatibility and all that.

Reply:
Just a note.... Images are never exposed as their alias, only using their main name. Aliases were added mainly with the backward compatibility in mind, so that you won't see them, when listing available image types, but if you use them to submit a compose, it will work just fine.

Architecture's `GetImageType()` checks also aliases
https://github.com/osbuild/osbuild-composer/blob/1e724be8e969b0d741f97d3bb7108e2218449de7/internal/distro/rhel85/distro.go#L124-L137

While `ListImageTypes()` does not

https://github.com/osbuild/osbuild-composer/blob/1e724be8e969b0d741f97d3bb7108e2218449de7/internal/distro/rhel85/distro.go#L115-L122

Reply:
So currently `edge-(commit|installer)` is exposed and translated to `rhel-edge-(commit|installer)`. This api should only ever support rhel-ga out of the rhels (so only rhel8.4 atm, maybe some day rhel 8.x and rhel 9.x at the same time), so I think  this all makes sense then? When rhel 8.5 goes ga we change the internal translation.